distributed/protocol/tests/test_numpy.py

```diff
@@ -138,24 +138,6 @@ def test_itemsize(dt, size):
     assert itemsize(np.dtype(dt)) == size
 
 
-@pytest.mark.skipif(sys.version_info[0] < 3,
-                    reason='numpy doesnt use memoryviews')
-def test_compress_numpy():
-    pytest.importorskip('lz4')
-    x = np.ones(10000000, dtype='i4')
-    frames = dumps({'x': to_serialize(x)})
-    assert sum(map(nbytes, frames)) < x.nbytes
-
-    header = msgpack.loads(frames[2], encoding='utf8', use_list=False)
-    try:
-        import blosc  # noqa: F401
-    except ImportError:
-        pass
-    else:
-        assert all(c == 'blosc' for c in
-                   header['headers'][('x',)]['compression'])
```

Comment:
This seems to be running for me

```
$ py.test distributed/protocol/tests/test_numpy.py::test_compress_numpy
============================= test session starts ==============================
platform linux -- Python 3.6.4, pytest-3.3.2, py-1.5.2, pluggy-0.6.0
rootdir: /home/mrocklin/workspace/distributed, inifile:
plugins: xdist-1.23.0, timeout-1.2.1, forked-0.2, cov-2.5.1
collected 1 item                                                               

distributed/protocol/tests/test_numpy.py .                               [100%]

=========================== 1 passed in 0.11 seconds ===========================
```

Reply:
> do you have blosc installed?

Yes, otherwise the test would have skipped.  Regardless, I think that the intent of this test is correct.  If it is failing for some reason then it would be good to dive into what is causing it to fail and resolve that.  Do you have any interest in figuring out why your data isn't compressing here?

Reply:
Sigh, I'm looking at the wrong code, I didn't have lz4 installed, so `test_compress_numpy` worked, but `test_compression_takes_advantage_of_itemsize` failed.  I will add an importskip for that, and add the test backin.

Sorry about that.